### PR TITLE
Adding -ShowPeoplePickerGroupSuggestionsForIB setting to Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by ZohrehHamrangsekachaie**
Adding ShowPeoplePickerGroupSuggestionsForIB setting. Please @samkabue review this change.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2101,6 +2102,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ShowPeoplePickerGroupSuggestionsForIB
+
+The ShowPeoplePickerGroupSuggestionsForIB setting (defaulted to false) will allow to show group suggestions for IB in the people picker.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2107]:
```suggestion
The ShowPeoplePickerGroupSuggestionsForIB setting (defaulted to false) allows showing group suggestions for information barriers (IBs) in the People Picker.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ZohrehHamrangsekachaie, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by dariomws — APPROVED** [review]
I'm ok after @JohanFreelancer9 suggestions. Thanks!

**Comment by get-itips** [comment]
Hello @ZohrehHamrangsekachaie 
Tested in Microsoft.Online.SharePoint.PowerShell version 16.0.22413.12000 and it's still not there, is this going to be released in next version soon?
Thank you

**Comment by ZohrehHamrangsekachaie** [comment]
> 

it has been released to SPDF as part of build16.0.22502.12000 at 4/29/2022 10:20 PM

**Review by samkabue — COMMENTED** [review]
@ZohrehHamrangsekachaie Please follow up with spacstrings to close on the string.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@ZohrehHamrangsekachaie

**Review by samkabue — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**